core/common/src/Localization.ts

```diff
@@ -110,7 +110,14 @@ export interface Localization {
  */
 export class EmptyLocalization implements Localization {
   public async initialize(): Promise<void> { }
-  public getLocalizedString(key: string | string[]): string { return typeof (key) == "string" ? key : key[0]; }
+  public getLocalizedString(key: string | string[]): string {
+    if (typeof (key) !== "string") {
+      key = key[0];
+    }
+    // Simulate correct and simple usage of i18next's translation function
+    // Namely, remove the leading namespace substring if there is one
+    return key.split(":", 2).pop()!;
```

Comment:
just curious why make this change?

Reply:
This change fixes a problem that appeared when switching to using EmptyLocalization for tests. 

When there is no value for the given key, i18next returns the key without the leading namespace. E.g., `getLocalizedString("myNamespace:myKey")` would return `"myKey"` if the key didn't exist. Before this change, EmptyLocalization would've returned `"myNamespace:myKey"` because it was simply returning the string. Some tests relied on how i18next returned a value, so I made EmptyLocalization simulate it.

Reply:
was this an issue with snapshots?